libs/ofxCv/src/Calibration.cpp

```diff
@@ -153,6 +153,105 @@ namespace ofxCv {
         updateUndistortion();
         ready = true;
     }
+    
+    void Calibration::loadLcp(string filename, float focalLength, bool absolute){
+        imagePoints.clear();
+        
+        // Load the XML
+        ofXml xml;
```

Comment:
Are you on latest openFrameworks? Its renamed (unsure when) to ofXml 
https://github.com/openframeworks/openFrameworks/blob/master/libs/openFrameworks/utils/ofXml.h